teams/templatetags/teams_tags.py

```diff
@@ -39,3 +39,10 @@ def divided_by(num, val):
 @register.filter
 def extract_userid(username):
     return username.split('_')[0]
+
+
+@register.filter
+def is_working_time(num):
+    _num = int(num.split(':')[0])
+    return 8 <= _num <= 20
+    return num
```

Comment:
Missing blank line at EOF

Reply:
Why 2x `return` statements at same indentation? Second one cannot be reached, no?

Reply:
Removed the second return and added blank line